clang/test/Preprocessor/pragma_mc_func.c

```diff
@@ -0,0 +1,23 @@
+// RUN: not %clang --target=powerpc64-ibm-aix -ferr-pragma-mc-func-aix -c -S \
+// RUN:   %s 2>&1 | FileCheck %s
```

Comment:
@qiongsiwu Driver tests usually use `-###` to still be able to exercise the test when the backend is not being built.

For example, this test is failing on the PS4 linux bot which doesn't build the PPC backend:
https://lab.llvm.org/buildbot/#/builders/144/builds/2931